kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/SchemaBuilder.kt

```diff
@@ -140,6 +141,12 @@ class SchemaBuilder internal constructor() {
         booleanScalar(T::class, block)
     }
 
+    fun extendedScalars() {
```

Comment:
Are there tests specifying that with not invoking this, the additional scalars are not present?

Reply:
I've seen all tests fail that I adjusted but it would make sense to have it validated explicitly. Will update but I have found some strange behavior with the implemented coercions unfortunately.

Reply:
This looks more complicated: #160

But I have at least added some validation